_posts/2023-05-08-benefits-of-speaking-djangocon.md

```diff
@@ -0,0 +1,40 @@
+---
+author: Jason Judkins
+category: General
+date: 2023-05-08 12:00:00
+image: /static/img/blog/cfp-announced-2023.png
+layout: post
+post_photo_alt: Speaker addressing a crowd at DjangoCon US 2022 in San Diego
+post_photo_url: /static/img/blog/speaking.jpg
+title: What are the Benefits of Speaking at DjangoCon?
+---
+
+ The DjangoCon US team would like to make sure you are aware of some of the benefits of giving a talk at DjangoCon US 2023.
+
+# Benefits you will receive for being a presenter:
+
+- **FREE! Ticket to DjancoCon US 2023.**
+
+- **Up to $500 in travel cost reimbursement!**
+
+- **A professionally produced video of your talk will be published on our YouTube channel.** (You may opt out of this.)
+
+- **Professional photographer on hand to photograph your talk.** (Also optional.)
+
+# Still on the fence deciding if you want to give a talk or not?
+
+- **Building your reputation:** Presenting at DjangoCon can establish you as an expert in the Django community, raising your profile and building your reputation.
+- **Networking:** Speaking at DjangoCon provides an opportunity to connect with other developers, designers, and enthusiasts in the Django community, allowing you to expand your network and make new connections.
+- **Career advancement:** Speaking at DjangoCon can help you advance your career by demonstrating your knowledge and expertise, which can lead to new job opportunities or promotions.
+- **Feedback and learning:** Presenting at DjangoCon gives you the opportunity to receive feedback from your peers, which can help you improve your skills and knowledge.
+- **Giving back to the community:** By sharing your knowledge and experience with others at DjangoCon, you are contributing to the growth and development of the Django community, which can be a rewarding experience in itself.
+- **Travel and conference experience:** Speaking at DjangoCon often includes travel to interesting locations and the chance to attend the conference and learn from other presenters, giving you a valuable conference experience that can help broaden your horizons.
+
+## Need Topic Ideas?
+
+Head over to our topic suggestion page! [Topic Suggestions](https://2023.djangocon.us/news/topic-suggestions-for-cfp/)
```

Comment:
I can't find an example of us doing this similarly, but I'd be curious if `{% post_url _posts/topic-suggestions-for-cfp %}` would work in place of this. It's a jekyll built-in, but I can't test it right now. The idea being that Jekyll will error if that URL/post changes.

Reply:
Updated it and it is working great!